src/lib/dynamic/keys.test.js

```diff
@@ -0,0 +1,51 @@
+const keys = require("./keys");
+
+describe("keys", () => {
+  describe("#singleKeysNotDynamic", () => {
+    context("with non-string key", () => {
+      it.skip("should return false", () => {
```

Comment:
Are we skipping these tests on purpose

Reply:
Yeah, they're being skipped because logically I think they should be returning false, but they're not - and the code is working fine in production.

So I've got them in here as a sort of documentation for future refactoring / understanding.